NewForce-Cohort-7/tabloidfullstack-bloodyscallwags
# Aa de branch

Description
Trello Ticket #19 https://trello.com/c/XUkEjn3z
Testing Instructions
git add and git commit whatever you're working on on your branch
git checkout main
git fetch --a
git checkout <ticket-name>
If you don't see the most recent version of my files, git pull origin <ticket-name>
If you haven't created the Tabloid MVC database, please do so.
Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-7/tabloidfullstack-bloodyscallwags/blob/main/SQL/01_Tabloid_Create_DB.sql)
Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-7/tabloidfullstack-bloodyscallwags/blob/main/SQL/02_Tabloid_Seed_Data.sql)
Click the execute with debugger solid green button to launch the app
In browser, please go to Login on the nav bar and log in with this email: [foo@bar.com](mailto:foo@bar.com)
This should redirect you back to the home page.
Go into Post and you will see the corresponding comments displayed.

Checklist:
[x] My code follows the style guidelines of this project
[x] I have performed a self-review of my code
[x] I have commented my code, particularly in hard-to-understand areas
[x] My changes generate no new warnings or errors
[x] I have added test instructions that prove my fix is effective or that my feature works
